setup.py

```diff
@@ -41,7 +38,7 @@
     author='Adam Tauber',
     author_email='asciimoo@gmail.com',
     license='GNU Affero General Public License',
-    packages=find_packages(exclude=["tests*", "searx_extra"]),
+    packages=find_packages(exclude=["tests*", "searx_extra", "searx.static.themes.simple.node_modules", "searx.static.themes.oscar.node_modules"]),
```

Comment:
Is this a part of this PR or is this a bugfix out of the scope of this PR?

Reply:
Nope, this is a leftover from a test.

Reply:
Fixed